core/threaded/reactor_threaded.c

```diff
@@ -1031,7 +1031,9 @@ void start_threads() {
     LF_PRINT_LOG("Starting %u worker threads.", _lf_number_of_workers);
     _lf_thread_ids = (lf_thread_t*)malloc(_lf_number_of_workers * sizeof(lf_thread_t));
     for (unsigned int i = 0; i < _lf_number_of_workers; i++) {
-        lf_thread_create(&_lf_thread_ids[i], worker, NULL);
+        if (lf_thread_create(&_lf_thread_ids[i], worker, NULL) != 0) {
```

Comment:
It seems that the check implemented in `lf_thread_create` should be done here, at the call site, before even invoking `lf_thread_create`...

Reply:
Since I have statically allocated the stacks for the threads I have to check it inside `lf_thread_create` to make sure that it is not called from somewhere else